setup/conda/conda_env.yml

```diff
@@ -0,0 +1,14 @@
+name: sc_env
+channels:
+  - conda-forge
+  - LiteX-Hub
```

Comment:
LiteX-Hub needs to be first otherwise you'll get the `netgen` package from `conda-forge` which is sadly totally unrelated: https://anaconda.org/conda-forge/netgen